### PR TITLE
solved day 8 with help of marco

### PR DESCRIPTION
**Pull request by thijsnissen**

### CODE REVIEW

File: 2018/src/test/scala/Test.scala [hunk 1]
```diff
@@ -25,13 +25,13 @@ class Test extends AnyFunSuite:
 		assertResult(3989)(actual = Day06.answerPart1)
 		assertResult(49715)(actual = Day06.answerPart2)
 
-	test("Day07"):
-		assertResult("IBJTUWGFKDNVEYAHOMPCQRLSZX")(actual = Day07.answerPart1)
-		assertResult(0)(actual = Day07.answerPart2)
+//	test("Day07"):
```
Comment (nmcb) [inline 2018/src/test/scala/Test.scala:28]: fyi - instead of commenting out a `test(..)` you can also `ignore(..)` it

**Review by nmcb — APPROVED** [review]
nice thijs - minor tip comment - otherwise lgtm :)

File: 2018/src/test/scala/Test.scala (hunk 1 above)
Comment (thijsnissen) [reply 2018/src/test/scala/Test.scala:28]: Ah Nice, I'm going to use that, thnx! How can I get the tests to run down here as well, before I merge? I've got 'Require status checks to pass before merging' enabled.

Comment (nmcb) [reply 2018/src/test/scala/Test.scala:28]: copy paste [this](https://github.com/nmcb/aoc2018/blob/main/.github/workflows/scala-build.yml) github action workflow and put it at the same location in your project <3

Comment (thijsnissen) [reply 2018/src/test/scala/Test.scala:28]: Hmm.. seems like I already [have](https://github.com/thijsnissen/advent-of-code/blob/main/.github/workflow/scala-build.yaml) that. Maybe a setting? We'll look into it next week 😃

Comment (nmcb) [reply 2018/src/test/scala/Test.scala:28]: or tomorrow ;)

( sleep well.. )

Comment (thijsnissen) [reply 2018/src/test/scala/Test.scala]: Fixed it, directory is supposed to be called 'workflows'. Mine was called 'workflow'. 👍🏻

**Pull request merged**